### PR TITLE
merge to production

### PR DESCRIPTION
**Pull request by prestonlimlianjie**

### CODE REVIEW

File: routes/sites.js [hunk 1]
```diff
@@ -59,24 +59,28 @@ router.get('/', async function(req, res, next) {
...
+      const resp = await axios.get(endpoint, {
+        params: {
+          per_page: 100,
+          page: pageCount,
+          sort: "full_name",
+        },
         headers: {
           Authorization: `token ${access_token}`,
-          "Content-Type": "application/json"
+          Accept: "application/vnd.github.baptiste-preview+json",
```
Comment (taufiqmmhd) [inline routes/sites.js]: is `baptiste_preview` necessary? That media type is for template repositories.
image

File: routes/sites.js [hunk 2]
```diff
@@ -85,7 +89,7 @@ router.get('/', async function(req, res, next) {
 
 
       siteNames= siteNames.concat(isomerRepos)
```
Comment (taufiqmmhd) [inline routes/sites.js]: put the space between the variable and the assignment operator please `siteNames=`

File: routes/sites.js (hunk 1 above)
Comment (kwajiehao) [reply routes/sites.js]: Not necessary, will remove in a separate PR

Comment (kwajiehao) [reply routes/sites.js]: resolved in #44

File: routes/sites.js (hunk 2 above)
Comment (kwajiehao) [reply routes/sites.js]: resolve in #44

**Pull request merged**